.circleci/config.yml

```diff
@@ -575,7 +575,7 @@ workflows:
                 - xenial
                 - bionic
               lite:
-                - "REDISAI_LITE=0 publish"
+                - "REDISAI_LITE=0 publish PUSH_GENERAL=1 OFFICIAL=1"
```

Comment:
why only in lite?
where is the branch/tag name instead of HEAD?

Reply:
When you operate on a branch, you have a legitimate name. Tags are detached HEADs, hence this change. This change lands here specifically, because we only push to dockerhub when LITE=0, as before. This can change....